### DISCUSSION TITLE
Web GUI Password Warning

### DISCUSSION THREAD

**Discussion by ajmoren** [MobiusSync/MobiusSync#205] [General]
I'm encountering a warning message every time I open the app. The warning states that I haven't set up a web GUI password or user, and it's recommending that I do so.

**Comment by SimonPickup** [comment]
That is very odd.

Is this a new installation?

Which version of Möbius Sync?

Which version of iOS?

Which phone model?

Have you done any advanced configuration?

**Reply by ajmoren** [reply]
It’s quite odd, but I was using the latest version of the free Möbius Sync app on my iPhone 14 Plus running iOS 18, which was fully updated. I hadn’t done any advanced configuration, yet sometimes when I opened the app, I would see the raw HTML — just the unstyled text — for a brief moment before everything finished loading and displayed normally, and this is where I would encounter the message. Since then, I’ve upgraded to the pro version, and while I have set up a username in the advanced configurations on the pro version, I haven’t encountered any issues.

**Reply by ajmoren** [reply]
I’ve encountered the same message on the Android and Windows versions of SyncThing. It warns about not setting up a username or password because hackers could access your web GUI and add their own devices and folders, potentially messing with your setup. Since I saw the raw HTML briefly when opening the app, I assume that Möbius Sync interacts with the web interface in a similar way, which might explain why this warning appears.

**Reply by SimonPickup** [reply]
Möbius Sync does not expose the web GUI and uses internal communication. Therefore you do not need to set a username or password and normally this warning is suppressed on Möbius Sync.

If you (or anyone else) sees this, please screenshot the warning and provide details of whether/how/when you set username/password.

**Reply by ajmoren** [reply]
Thank you for the response and the clarification. I appreciate the insight on how Möbius Sync handles internal communication and suppresses the warning. If I encounter this warning again, I’ll be sure to take a screenshot and provide the details regarding the username/password settings. Thanks again for your help!
